monai/bundle/scripts.py

```diff
@@ -52,12 +55,61 @@ def _update_args(args: Optional[Union[str, Dict]] = None, ignore_none: bool = Tr
 
 
 def _log_input_summary(tag: str, args: Dict):
+    """
```

Comment:
The private functions might change

Reply:
Sure, removed in the latest commit.

Thanks.